youtube_dl/extractor/svt.py

```diff
@@ -225,7 +225,8 @@ def _real_extract(self, url):
                 return info_dict
 
         svt_id = self._search_regex(
-            r'<video[^>]+data-video-id=["\']([\da-zA-Z-]+)',
+            (r'<video[^>]+data-video-id=["\']([\da-zA-Z-]+)',
+             r'"content":\{"id":"([\da-zA-Z-]+)'),
```

Comment:
Maybe using the already parsed json will be stabler. Something like this:
```patch
diff --git a/youtube_dl/extractor/svt.py b/youtube_dl/extractor/svt.py
index e12389cad..d446bc114 100644
--- a/youtube_dl/extractor/svt.py
+++ b/youtube_dl/extractor/svt.py
@@ -224,9 +224,11 @@ class SVTPlayIE(SVTPlayBaseIE):
                 self._adjust_title(info_dict)
                 return info_dict

-        svt_id = self._search_regex(
-            r'<video[^>]+data-video-id=["\']([\da-zA-Z-]+)',
-            webpage, 'video id')
+        svt_id = try_get(data, lambda x: x['statistics']['dataLake']['content']['id'], compat_str)
+        if not svt_id:
+            svt_id = self._search_regex(
+                r'<video[^>]+data-video-id=["\']([\da-zA-Z-]+)',
+                webpage, 'video id')

         return self._extract_by_video_id(svt_id, webpage)
```

Could possible instead or also use `svtId ` or `videoSvtId ` in the `__svtplay_apollo` json structure?